DTML.EduBot/Dialogs/NoneDialog.cs

```diff
@@ -22,10 +22,10 @@ public partial class ChitChatDialog : QnaLuisDialog<object>
             await context.PostAsync(BotPersonality.BotResponseUnrecognizedIntent);
         }
 
-       [LuisIntent("Gibberish")]
-        public async Task HandleGibberish(IDialogContext context, LuisResult result)
-        {
-            await context.SayAsync(BotPersonality.BotResponseToGibberish, BotPersonality.BotResponseToGibberish);
-        }
+       //[LuisIntent("Gibberish")]
```

Comment:
a part of the code that is commented out is called "Dead Code". I assume this was left over while testing the code. Dead code should be removed before code check in.